tests/nn/data_parallel/test_fsdp_regnet.py

```diff
@@ -350,7 +352,7 @@ def dump(d):
 @pytest.mark.parametrize("flatten", ["flatten", "no_flatten"])
 @pytest.mark.parametrize("sync_bn", ["none", "pytorch"])
 def test_regnet(temp_files, ddp_ref, precision, flatten, sync_bn):
-    if torch_version() < (1, 6, 0):
+    if torch_version() < (1, 9, 0):
         pytest.skip("older pytorch doesn't support reduce_scatter")
```

Comment:
this skip msg is not longer right after the change?

Reply:
modified the skip message appropriately.